drools-compiler/src/main/java/org/drools/compiler/kie/builder/impl/ClasspathKieProject.java

```diff
@@ -500,6 +512,28 @@ private static String getPathForVFS(URL url) {
                 path = path.substring( 0, path.length() - ("/" + KieModuleModelImpl.KMODULE_JAR_PATH).length() );
             }
 
+            if (!Files.exists(Paths.get(path))) {
+                log.info("path : " + path + " --- does not exist");
+                log.info("fallback for EAP 7.14.5");
+                try {
+                    Method m3 = Class.forName("org.jboss.vfs.VFS").getDeclaredMethod("getMount", Class.forName("org.jboss.vfs.VirtualFile"));
+                    m3.setAccessible(true);
+                    Method m4 = Class.forName("org.jboss.vfs.VFS$Mount").getDeclaredMethod("getFileSystem");
+                    m4.setAccessible(true);
+                    Method m5 = Class.forName("org.jboss.vfs.spi.FileSystem").getMethod("getMountSource");
+
+                    Object virtualFile = m2.invoke(null, url.toURI());
+                    Object mount = m3.invoke(null, virtualFile);
+                    Object fileSystem = m4.invoke(mount);
+                    log.info("fileSystem : " + fileSystem);
+                    File mountSource = (File) m5.invoke(fileSystem);
+                    log.info("mountSource : " + mountSource);
+                    path = mountSource.getPath();
+                } catch (Exception e) {
+                    e.printStackTrace(System.out);
+                }
+            }
```

Comment:
This is the main part of the (temporary) fix. I was not able to find public methods to get the physical file (e.g. `kie-server-integ-tests-drools/target/cargo/configurations/wildfly23x/tmp/vfs/temp/tempc483d0975993750a/content-c8ee66a0826ffc48/WEB-INF/lib/kie-pmml-7.75.0-SNAPSHOT.jar`). So used `setAccessible(true)` for package-private methods.